src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
+  {#name, nparam, \
+   params, \
+   (GVarFunc)name, \
+   srcfile ":Func" #name }
```

Comment:
BTW, one could also consider removing the `srcfile` argument, and instead using `__FILE__`. Which may produce more unwieldily names, but it also benefits:
* You can't forget to update the name when renaming files (this happened to me several times)
* You can't have typos in the filename (this was the case in the GAP kernel)
* Since the path is relative to the directory `make` run in, for me it would actually be helpful, because I cold cmd-click on such paths to open the relevant files in my editor ;-).